### ISSUE TITLE
Image with link from draw.io.

### ISSUE THREAD

**Issue by Technik-J** [BookStackApp/BookStack#1548]
**Describe the feature you'd like**
When you insert an image, this image also have a link attached to it, so you can click on it and it will open in full view. This happens with all images, except with the ones that made with draw.io. For this images you have to do "Right click -> View Image". Is it possible to also attach link to this kind of images, so it will open with one click?

**Describe the benefits this feature would bring to BookStack users**
Less clicks and convenience.

**Comment by simchanu29** [comment]
It seems the same issue as #1170 
It would be nice to have this feature.

**Comment by ssddanbrown** [comment]
Hey @simchanu29, #1170 refers to links embedded within the drawing. This issue is about wrapping a link around the inserted drawing so you can easily view the full version when clicked, as occurs with standard images.

**Comment by mv-yurchenko** [comment]
@ssddanbrown Any info about release of this feature?

**Comment by ssddanbrown** [comment]
@mv-yurchenko Nope, adds complication adding in an official way (Where I'd want to do this properly) and has potential considerations/conflicts with other drawing requests, but here's a way to essentially add this functionality to your instance right now, by adding the below to the "Custom HTML Head Content" customization setting:

```html
<script type="module">
    const drawings = document.querySelectorAll('.page-content [drawio-diagram] img');
    for (const drawing of drawings) {
        drawing.addEventListener('click', event => window.open(event.target.src, '_target'));
    }
</script>
<style>
    .page-content [drawio-diagram] img {
        cursor: pointer;
    }
</style>
```

**Comment by mv-yurchenko** [comment]
It works for me, thanks a lot!

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/1548
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
